src/ExportGenerators.mss

```diff
@@ -1112,6 +1112,17 @@ function GenerateNote (nobj) {
     libmei.AddAttribute(n, 'dur.ges', gesdur & 'p');
     libmei.AddAttribute(n, 'dots', meidur[1]);
 
+    if (nobj.Dx != 0)
+    {
+        libmei.AddAttribute(n, 'ho', ConvertOffsetsToMEI(nobj.Dx));
+    }
+
+    if (nobj.Color != 0)
```

Comment:
Makes sense!

Reply:
Just ran the tests (I highly recommend running tests and writing tests for new features).  This needs to be `nobj.ParentNoteRest` instead of `bobj.ParentNoteRest`.

Reply:
I should have looked myself. I'll add a test file eventually. If you got one already, please add it by yourself.